### ISSUE TITLE
cannot use foundation sites 6.2.1 with meteor 1.2.1

### ISSUE THREAD

**Issue by SierraGolf** [zurb/foundation-sites#8721]
**How can we reproduce this bug?**
1. `meteor create --release 1.2.1 foundation-test`
2. `cd foundation-test`
3. `meteor add zurb:foundation-sites`
4. `meteor`

**What did you expect to happen?**
The application to start.

**What happened instead?**

```

=> Started proxy.
=> Meteor 1.3.2.4 is available. Update this project with 'meteor update'.
=> Started MongoDB.
packages/zurb_foundation-sites/test/javascript/lib/mocha.css: warn: @charset rules in this file will be ignored as UTF-8 is the only encoding supported
=> Errors prevented startup:

   While processing files with ecmascript (for target web.browser):
   vendor/jquery/src/intro.js:45: vendor/jquery/src/intro.js: Unexpected token (45:0)
   vendor/jquery/src/outro.js:2: vendor/jquery/src/outro.js: Unexpected token (2:0)

   While processing files with fourseven:scss (for target web.browser):
   /packages/zurb_foundation-sites/docs/assets/scss/docs.scss: Scss compiler error: File to import:
   {zurb:foundation-sites}/docs/assets/scss/settings/settings not found in file:
   /foundation-test/{zurb:foundation-sites}/docs/assets/scss/docs.scss


   While computing assigned variables:
   /packages/zurb_foundation-sites/vendor/jquery/src/intro.js: Unexpected token (47:1)
   /packages/zurb_foundation-sites/vendor/jquery/src/outro.js: Unexpected token (2:1)

=> Your application has errors. Waiting for file change.
```

This does not happen if I try it with meteor 1.3.2.4.

**Comment by kevster** [comment]
I just upgraded to Meteor 1.3.3 with Foundation Sites 6.2.3 (latest releases at time of writing) and I'm getting the same error as described by @SierraGolf, also using same reproduction method.

**Comment by nowinskp** [comment]
Have the same problem, except for me it's only:
`While computing assigned variables:
    /packages/zurb_foundation-sites/vendor/jquery/src/intro.js: Unexpected token (45:0)
    /packages/zurb_foundation-sites/vendor/jquery/src/outro.js: Unexpected token (2:0)`

I had to downgrade Meteor to 1.3.2.4 to make it work.

**Comment by slissner** [comment]
+1

**Comment by lerayj** [comment]
Same as @nowinskp

**Comment by scottwillman** [comment]
Same as @nowinskp on 1.3.3.1

**Comment by Bernardao** [comment]
+1

**Comment by summanerd** [comment]
+1

**Comment by summanerd** [comment]
I downgraded meteor to 1.3.2.4 to get around this issue. jQuery 1.11.9 seems to be the issue and a few of the packages in meteor rely on it in version 1.3.3.1. Using foundation package version 6.2.3.

**Comment by DanielRuf** [comment]
It seems this is only an issue with meteor and jQuery so we have no control over this.

Also there might be new meteor versions which do not have this issue.

**Comment by DanielRuf** [comment]
@ncoden can I assign this to you as you will address some Meteor related stuff?

**Comment by DanielRuf** [comment]
Bump =)

**Comment by ncoden** [comment]
Yes =) At least for a simple check that `v6.5.0` will work correctly on Meteor.

**Comment by DanielRuf** [comment]
We've released v6.5 for Meteor. If the issue still exists there, please let us know.
